source/release-notes/v1.7-release-notes.rst

```diff
@@ -76,6 +69,17 @@ Upgrading from v1.6
       sudo /opt/ood/ood-portal-generator/sbin/update_ood_portal --force
       sudo systemctl try-restart httpd24-httpd.service httpd24-htcacheclean.service
 
+   .. warning:
```

Comment:
I think this ` .. warning:` needs to have 1 more colon, ` .. warning::` . This warning block doesn't appear.